internal/js/summary.js

```diff
@@ -839,7 +839,7 @@ function renderThresholdResults(
 			: formatter.decorate(failMark, 'red');
 
 		const sourceText = formatter.decorate(
-			`'${threshold.source}'`,
```

Comment:
This isn't directly related, but it's so small that I prefer to add it as part of this PR. It's not a huge deal, but I think it's nice if we trim the threshold's source before displaying it in the summary.

Reply:
> If the source includes spaces why we don't interpret it as intentional?

I'm a bit confused now; is there any reason for doing so? 

To me, it feels like when you mistakenly add a leading or trailing whitespace in an input designed to write your name or similar.

Looking at the threshold syntax expression ([ref](https://grafana.com/docs/k6/latest/using-k6/thresholds/#threshold-expression-syntax)), I can see for instance users adding (or not) whitespaces before/after the `<operator>`, for readability, but not why doing so at the beginning/end of the threshold definition.